fabric-serialization-api-v1/src/testmod/java/net/fabricmc/fabric/test/serialization/DelegateWriteView.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.serialization;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.MapCodec;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.storage.WriteView;
+
+/**
+ * A delegating WriteView, used to force usage of fallback implementation of FabricWriteView
+ */
+public record DelegateWriteView(WriteView view) implements WriteView {
+	@Override
+	public <T> void put(String key, Codec<T> codec, T value) {
+		view.put(key, codec, value);
+	}
+
+	@Override
+	public <T> void putNullable(String key, Codec<T> codec, @Nullable T value) {
+		view.put(key, codec, value);
```

Comment:
should this not delegate to the same method on `view`?
```suggestion
		view.putNullable(key, codec, value);
```